corehq/ex-submodules/casexml/apps/case/models.py

```diff
@@ -822,10 +822,13 @@ def force_save(self, **params):
             self._doc["_rev"] = conflict._rev
             self.force_save()
 
-    def to_xml(self, version):
+    def to_xml(self, version, include_case_on_closed=False):
         from xml.etree import ElementTree
         if self.closed:
-            elem = get_case_element(self, ('close'), version)
+            if include_case_on_closed:
+                elem = get_case_element(self, ('close', 'create', 'update'), version)
```

Comment:
seems worth using the same order as the spec (so `create` then `update` then `close`)